interview-questions.md

```diff
@@ -6,50 +6,91 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
+
+
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer:  
+    //JSX is an extension used in REACT where we can inject JS into HTML element tags.
+    //A difference between HTML & JSX syntax is:
+        -HTML: <div></div>
+        -JSX: <div />
 
 Researched answer:
+    //JSX is a JavaScript syntax extension. It helps REACT describe what and how a UI should look and feel. You can inject JS into HTML tags by using curly braces. JSX creates elements which can be used to render to the DOM, infusing logic and UI to be handled with events.
+
+
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
 Your answer:
+    //Yarn is a distributor that provides bundled packages to help quickly setup a developer environment inside an editor for JavaScript.
+    //The files that are modified in React when yarn is run is that it adds:
+        //node_modules
+        //yarn.lock
+        //package.json
 
 Researched answer:
+    //Yarn is a package manager that also is a project manager. It was developed in 2016 by Facebook for the Node.js JavaScript environment. It is an alternative to npm package manager.
+
+
 
 3. What is an anonymous function in JavaScript?
 
 Your answer:
+    //An anonymous function is a function with no name. It is displayed as below:
+        -->         () => {}
 
 Researched answer:
+    //Anonymous function is a nameless function with its only purpose being accepting multiple inputs and returning one output for a variable to do a function handle. It can be used as an argument for another function.
+
+
 
 4. What is the difference between state values and props in React?
 
 Your answer:
+    //State values in REACT are local data that is stored within the component.
+    //Props are used to send data to a component to pass an argument.
 
 Researched answer:
+    //Both states and props are JavaScript objects. Both hold information that can determine the way something can render, they are different. Props get passsed to the component similar to a function getting a paramater as an input. While, state is controlled within the component it resides in.
+
+
 
 5. What is the DOM?
 
 Your answer:
+    //Document-Object-Model
```

Comment:
While correct, this answer in an interview is not very conversational and will likely not impress your interviewer.  Make sure to expand on your answers.